Signal/src/ViewControllers/FingerprintViewScanController.m

```diff
@@ -82,7 +82,7 @@ - (void)createViews
     footer.backgroundColor = darkGrey;
     [self.view addSubview:footer];
     [footer autoPinWidthToSuperview];
-    [footer autoPinToBottomLayoutGuideOfViewController:self withInset:0];
+    [footer autoPinEdgeToSuperviewEdge:ALEdgeBottom];
```

Comment:
Please ignore the fact that these screenshots were taken "mid-transition" (hence the previous view being visible on the left edge and the navbar having content in transitition.)

**Before**

<img width="641" alt="screen shot 2017-11-08 at 9 11 24 am" src="https://user-images.githubusercontent.com/217057/32553405-d9314854-c464-11e7-82e7-d9c0787a6ab6.png">

**After**
<img width="545" alt="screen shot 2017-11-08 at 8 52 48 am" src="https://user-images.githubusercontent.com/217057/32553218-38844cc6-c464-11e7-9b5e-0013cfa13119.png">

**Regression Check**
<img width="559" alt="screen shot 2017-11-08 at 8 52 01 am" src="https://user-images.githubusercontent.com/217057/32553217-38797a12-c464-11e7-9bb6-2d82b070d783.png">